src/runtime/pay-client-test.js

```diff
@@ -378,17 +413,29 @@ describes.realWin('PayClientBindingSwg', {}, env => {
 });
 
 describes.realWin('PayClientBindingPayjs', {}, env => {
+  let deps;
   let win;
+  let eventManager;
+  let globalDoc;
   let resultStub;
   let activityPorts;
   let payClient;
   let payClientStubs;
+  let pageConfig;
+  let analyticsService, analyticsMock;
+  let dialogManager, dialogManagerMock;
   let redirectVerifierHelperStubs, redirectVerifierHelperResults;
   let responseHandler;
+  let googleTransactionId;
 
   beforeEach(() => {
+    deps = new DepsDef();
     win = env.win;
     activityPorts = new ActivityPorts(win);
+    sandbox.stub(deps, 'win').callsFake(() => win);
+
+    googleTransactionId = 'GOOGLE_TRANSACTION_ID';
```

Comment:
NIT: My opinion is that it would be cleaner to do this once at the top of the file, just after the includes:

const GOOGLE_TRANSACTION_ID = 'ANY_VALUE_YOU_WANT';

Reply:
ah gotcha. I'll update it. Thanks!